README.md

```diff
@@ -1 +1,33 @@
-# example-backend-ruby-sinatra
+
+# Example Shunter Application Back End (Ruby, Sinatra)
+
+A Ruby and Sinatra-based back end for the [Shunter example application](https://github.com/shunterjs/example).
+
+
+## Requirements
+
+`Ruby 2.2.2+`
```

Comment:
It turns out I need 1.9+ because of the Hash syntax, so I will add a `ruby.version.example` :+1: